Jmg branch

jmarcusgill opened the pull request:
@GAylabea, @abidina 

The assignment is complete. If you look at the 2nd and 3rd cards, the spacing after the product description is a little off, but the flex is flexing and it looks good. If I have time I will figure out the spacing issue, or if y'all have pointers let me know!

jmarcusgill committed:
added boilerplate html

jmarcusgill committed:
linked css and changed bground color

jmarcusgill committed:
everything is done, the 2nd and 3rd cards spacing is a bit off but it works.

GAylabea commented:
This looks great and you are at MVP according to the written specifications. Some comments - although it doesn't say so in the specifications, the example implies that the 4th card should align left - it doesn't on my computer. It also implies that each card should have a unique product title and information. It's nit-picky and it doesn't matter now but in the future when/if you copy/paste (even into your own code) it can make a difference if you don't edit carefully. It's a lesson I've learned (and re-learned and re-learned and re-learned...) the hard way. 
Anyway, can we see what it looks like on your computer before merging?

jmarcusgill committed:
fixed repeating titles

jmarcusgill committed:
fixed issue with cards not aligning to left

jmarcusgill commented:
@GAylabea,

fixed the issue with the cards not lining up with the left side of the screen. Also fixed the repeating "Yellow Dunk Card" title. It looks a lot better!
